src/json-schema/partial/map_state.json

```diff
@@ -246,22 +217,14 @@
                                                 "STANDARD"
                                             ]
                                         }
-                                    },
-                                    "required": [
-                                        "Mode", 
-                                        "ExecutionType"
-                                    ]
+                                    }
                                 }    
-                            },                   
-                            "required": [
-                                "ProcessorConfig"
-                            ]
+                            }
                         },
-                        "ToleratedFailurePercentage": {
```

Comment:
`ToleratedFailurePercentage` was removed in the last commit, can you add it back?

Reply:
It can be a floating point number which isn't supported in the plugin yet. We'll have to build that out which can be done when we add type checking.